### PR TITLE
Ensure non-zero send value in prompt.py

### PR DESCRIPTION
**Pull request by jseagrave21**
**What current issue(s) does this address, or what feature is it adding?**
Ensures a user cannot send 0 of an asset

Addresses Issue #606 

NOTE: This PR's test relies on `BlockchainFixtureTestCase` through `WalletFixtureTestCase`

**How did you solve this problem?**
Trial and Error

**How did you make sure your solution works?**
testing on privatenet and unittest

**Are there any special changes in the code that we should be aware of?**
No

**Please check the following, if applicable:**

- [x] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?
- [x] Are you making a PR to a feature branch or development rather than master?
- [x] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

File: neo/Prompt/Commands/Send.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ def construct_and_send(prompter, wallet, arguments, prompt_password=True):
         address_to = get_arg(arguments, 1)
         amount = get_arg(arguments, 2)
 
+        if int(float(amount)) == 0:
```
Comment (ixje) [inline neo/Prompt/Commands/Send.py]: just `float(amount)` is enough, no need to `int()` as well
```
>>> float("0.0") == 0
True
>>> float("0") == 0
True
```

File: neo/Prompt/Commands/Send.py [hunk 2]
```diff
@@ -31,6 +31,10 @@ def construct_and_send(prompter, wallet, arguments, prompt_password=True):
...
+            print("invalid amount format")
```
Comment (ixje) [inline neo/Prompt/Commands/Send.py]: can we make the message more direct like; "amount cannot be 0" or something along those lines. As this message might suggest "0" is not valid and should be 0.0 or something.

File: neo/Prompt/Commands/Send.py (hunk 1 above)
Comment (jseagrave21) [reply neo/Prompt/Commands/Send.py]: Okay, thank you for the feedback! I really appreciate it!

File: neo/Prompt/Commands/Send.py (hunk 2 above)
Comment (jseagrave21) [reply neo/Prompt/Commands/Send.py]: Absolutely. Thank you for the feedback!

**Comment by ixje** [comment]
Do not merge until #478 is finished and merged

**Comment by ixje** [comment]
great! Yes I like your suggestion but you can put that in a separate PR.

**Pull request merged**